packages/dropdowns/demo/menu.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { useArgs } from '@storybook/client-api';
+import Downshift from 'downshift';
+import {
+  AddItem,
+  Dropdown,
+  HeaderIcon,
+  HeaderItem,
+  Item,
+  ItemMeta,
+  MediaBody,
+  MediaFigure,
+  MediaItem,
+  Menu,
+  NextItem,
+  PreviousItem,
+  Separator,
+  Trigger
+} from '@zendeskgarden/react-dropdowns';
+import { TriggerStory } from './stories/TriggerStory';
+import { MENU_ITEMS as ITEMS } from './stories/data';
+
+<Meta
+  title="Packages/Dropdowns/Menu"
+  component={Menu}
+  subcomponents={{
+    AddItem,
+    Dropdown,
+    HeaderIcon,
+    HeaderItem,
+    Item,
+    ItemMeta,
+    MediaBody,
+    MediaFigure,
+    MediaItem,
+    Menu,
+    NextItem,
+    PreviousItem,
+    Separator,
+    Trigger
+  }}
+/>
+
+# API
+
+<ArgsTable />
+
+# Demo
+
+<Canvas>
+  <Story
+    name="Menu"
+    args={{
+      eventsEnabled: true,
+      isAnimated: true,
+      items: ITEMS,
+      selectedItems: [ITEMS[5]],
+      hasMedia: false,
+      isOpen: false
+    }}
+    argTypes={{
+      downshiftProps: { control: 'object', table: { category: 'Dropdown' } },
+      highlightedIndex: { control: 'number', table: { category: 'Dropdown' } },
+      isOpen: { table: { category: 'Dropdown' } },
+      selectedItems: { table: { category: 'Dropdown' } },
+      disabled: { control: 'boolean', table: { category: 'Item' } },
+      isDanger: { control: 'boolean', table: { category: 'Item' } },
+      hasMedia: { table: { category: 'Story' } },
+      items: { name: 'Item[]', table: { category: 'Story' } }
+    }}
+    parameters={{
+      design: [
+        {
+          name: 'Menu',
+          allowFullscreen: true,
+          type: 'figma',
+          url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=254%3A1241'
+        },
+        {
+          name: 'Item',
+          allowFullscreen: true,
+          type: 'figma',
+          url: 'https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=1488%3A28208'
+        }
+      ]
+    }}
+  >
+    {args => {
+      const updateArgs = useArgs()[1];
+      const handleSelect = selectedItems => updateArgs({ selectedItems });
+      const handleStateChange = changes =>
+        changes.hasOwnProperty('isOpen') &&
+        changes.type !== Downshift.stateChangeTypes.blurInput /*account for Storybook control*/ &&
+        updateArgs({ isOpen: changes.isOpen });
+      return <TriggerStory {...args} onSelect={handleSelect} onStateChange={handleStateChange} />;
```

Comment:
It looks like there is an issue with the chevron when the nested menu is expanded.

<img width="279" alt="Screen Shot 2022-01-04 at 2 45 54 PM" src="https://user-images.githubusercontent.com/1811365/148134450-293a3f63-5b4b-40fa-a373-7a8ef99195d0.png">

Reply:
Yes. This is just one of a number of issues where a finessed storybook was hiding component cases that can happen IRL. In this case a `NextItem` that is set as the `Dropdown` `selectedItem` styles with a checkmark. The component styling should be preventing this from happening. I am filing a list of follow-on bugs to be addressed after this PR.